src/platforms/hosted/bmp_libusb.c

```diff
@@ -413,6 +413,7 @@ static void check_cmsis_interface_type(libusb_device *const device, bmda_probe_s
 
 			/* Check if it's a CMSIS-DAP v2 interface */
 			if (descriptor->bInterfaceClass == 0xffU && descriptor->bNumEndpoints == 2U) {
+				break;
```

Comment:
I am not sure if I see it right but it seems to just skip the rest of the code block by breaking here.

Reply:
Hah, that's a rebase fail - good spot! We've removed the offending commit as that would have introduced a bug (never allowing CMSIS-DAP v2 adaptors).